packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropsWithChildren } from "react";
+import classNames from "classnames";
+import { useChildComponent } from "@jobber/hooks/useChildComponent";
+import styles from "../../Chip.css";
+import { Icon } from "../../../Icon";
+
+interface ChipSuffixProps extends PropsWithChildren {
+  className?: string;
+}
+
+export function ChipSuffix({ children, className }: ChipSuffixProps) {
+  let singleChild = useChildComponent(children, d => d.type === Icon);
+  if (!["cross", "add", "checkmark"].includes(singleChild?.props?.name)) {
```

Comment:
Are these allowed icons? Can we extract this arrow into a more readable constant so we know what it is about?